packages/beacon-node/src/chain/blocks/verifyBlocksDataAvailability.ts

```diff
@@ -92,7 +106,16 @@ async function maybeValidateBlobs(
       const skipProofsCheck = opts.validBlobSidecars === BlobSidecarValidation.Individual;
       validateBlobSidecars(blockSlot, beaconBlockRoot, blobKzgCommitments, blobs, {skipProofsCheck});
 
-      return DataAvailableStatus.available;
+      const availableBlockInput = getBlockInput.postDeneb(
+        chain.config,
+        blockInput.block,
+        blockInput.source,
+        blobs,
+        BlobsSource.gossip,
```

Comment:
the hard code does not look right here, this should be from the `blockInput.availabilityPromise`?

Reply:
ahh thanks for noticing